packages/shorebird_cli/lib/src/shorebird_artifacts.dart

```diff
@@ -10,14 +10,20 @@ import 'package:shorebird_cli/src/shorebird_env.dart';
 
 /// All Shorebird artifacts used explicitly by Shorebird.
 enum ShorebirdArtifact {
-  /// The analyze_snapshot executable.
-  analyzeSnapshot,
+  /// The iOS analyze_snapshot executable.
+  analyzeSnapshotIos,
+
+  /// The macOS analyze_snapshot executable.
+  analyzeSnapshotMacOS,
```

Comment:
I guess android doesn't use analyze_snapshot or gen_snapshot thus this is the first time these have had a paltform component?

Reply:
Yep!